cfme/cloud/keypairs.py

```diff
@@ -208,6 +208,16 @@ def create(self, name, provider, public_key=None, cancel=False):
         view.flash.assert_success_message(flash_message)
         return self.instantiate(name, provider, public_key=public_key)
 
+    def all(self):
+        view = navigate_to(self, 'All')
+        result = []
+        for _ in view.entities.paginator.pages():
```

Comment:
Understood why you used paginator on the other items, but here you're only using the names from the table. I think you can safely use `view.entityes.all_entity_names` here to not have to surf pages.  Then this can even be squashed into a list comprehension if you want.